pkg/syncer/cnsoperator/controller/cnsnodevmbatchattachment/cnsnodevmbatchattachment_helper.go

```diff
@@ -552,14 +554,34 @@ func constructBatchAttachRequest(ctx context.Context,
 			isPvcEncrypted := isPvcEncrypted(pvcObj.Annotations)
 			log.Infof("PVC %s has encryption enabled: %t", pvcName, isPvcEncrypted)
 
+			// Get BackingType from PVC annotation, if not available query from VirtualDiskManager
+			backingType := pvcObj.GetAnnotations()[common.AnnKeyBackingDiskType]
+			if backingType == "" {
+				log.Infof("BackingType annotation not found on PVC %s, querying from VirtualDiskManager", pvcName)
+				queriedBackingType, queryErr := volumeManager.QueryBackingTypeFromVirtualDiskInfo(ctx, volumeID)
+				if queryErr != nil {
+					log.With("pvc", pvcName).With("namespace", instance.Namespace).Error(queryErr)
+					return pvcsInSpec, volumeIdsInSpec, batchAttachRequest, queryErr
+				}
+				backingType = queriedBackingType
+				log.Infof("Successfully retrieved BackingType %s for PVC %s from VirtualDiskManager",
```

Comment:
Can you log the instance, that way we know which reconcile this specific log is a part of